tpl/cast/cast_test.go

```diff
@@ -81,3 +81,37 @@ func TestToString(t *testing.T) {
 		assert.Equal(t, test.expect, result, errMsg)
 	}
 }
+
+func TestToFloat(t *testing.T) {
+	t.Parallel()
+
+	ns := New()
+
+	for i, test := range []struct {
+		v      interface{}
+		expect interface{}
+	}{
+		{"1", 1.0},
```

Comment:
These tests assumes that the input is strings only. I suspect your implementation will fail on `2, int64(2), float64(2)` etc.

Reply:
I've pushed additional additional test cases, they all pass.

Reply:
Yea, I see now -- I read your original code slightly wrong, but it helped with the test cases.